### PR TITLE
packages: add yip back

### PR DESCRIPTION
**Pull request by Vicente-Cheng**
we still need yip for the installer with network setup manually.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -45,6 +45,10 @@ RUN curl -o /usr/bin/rancherd -sfL "https://github.com/rancher/rancherd/releases
 ARG VIRTCTL_VERSION=v0.55.2
 RUN curl -o /usr/bin/virtctl -sfL "https://github.com/kubevirt/kubevirt/releases/download/${VIRTCTL_VERSION}/virtctl-${VIRTCTL_VERSION}-linux-amd64" && chmod 0755 /usr/bin/virtctl
 
+# Download yip
+ARG YIP_VERSION=v1.0.0
```
Comment (guangbochen) [inline Dockerfile:49]: wondering why we skip install via luet?

**Review by bk201 — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (bk201) [reply Dockerfile:49]: New versions are not published to luet repo anymore.

Comment (Vicente-Cheng) [reply Dockerfile:49]: Yap, for the short term, we plan to download directly.
For the long term, we may try to maintain our package list on OBS.

**Review by guangbochen — APPROVED** [review]
LGTM, thanks.

**Pull request merged**